TESTS/usb_device/basic/USBEndpointTester.cpp

```diff
@@ -173,7 +171,6 @@ USBEndpointTester::USBEndpointTester(USBPhy *phy, uint16_t vendor_id, uint16_t p
         MBED_ASSERT(_endpoint_buffs[i] != NULL);
     }
     MBED_ASSERT(resolver.valid());
-    queue = mbed::mbed_highprio_event_queue();
```

Comment:
by removing the queue here, the test does not change - it has not been used at all?

Reply:
yes. This queue initially added by Russ to create a utility function but the scope of this work became obsolete. This queue is not used any of the test cases on full profile.